06_neighborhoods/Transportation/transit_trips_cost_county_subgroups.R

```diff
@@ -0,0 +1,740 @@
+transit_cost_tracts_2019$GEOID <- paste0(transit_cost_tracts_2019$state, transit_cost_tracts_2019$county, transit_cost_tracts_2019$tract)
+transit_trips_tracts_2015$GEOID <- paste0(transit_trips_tracts_2015$state, transit_trips_tracts_2015$county, transit_trips_tracts_2015$tract)
+transit_trips_tracts_2019$GEOID <- paste0(transit_trips_tracts_2019$state, transit_trips_tracts_2019$county, transit_trips_tracts_2019$tract)
+
+
+# merge population data with transit data files (4 of them)
+############################################################################
+# FIRST ONE - Cost in 2015
+############################################################################
+cost_data_2015 <- left_join(acs_tract_pop15, transit_cost_tracts_2015, by = "GEOID")
+#Test with anti_join to make sure it worked properly
+#stopifnot(
+#  anti_join(acs_tract_pop15, transit_cost_tracts_2015, by = "GEOID") %>%
+#    nrow() == 0
+#)
+# Error - therefore, check how many missing cost values for observations where we have population
+#missing_count <- sum(is.na(cost_data_2015$t_80ami))
+#cat("Number of missing values for t_80ami:", missing_count, "\n")
+# 810 missing values
+
+############################################################################
+# SECOND ONE - Cost in 2019
+############################################################################
+cost_data_2019 <- left_join(acs_tract_pop19, transit_cost_tracts_2019, by = "GEOID")
+#Test with anti_join to make sure it worked properly
+stopifnot(
+  anti_join(acs_tract_pop19, transit_cost_tracts_2019, by = "GEOID") %>%
+    nrow() == 0
+)
+# Error - therefore, check how many missing cost values for observations where we have population
+missing_count <- sum(is.na(cost_data_2019$t_80ami))
+cat("Number of missing values for t_80ami:", missing_count, "\n")
+# 831 missing values
+
+############################################################################
+# THIRD ONE - Trips in 2015
+############################################################################
+trips_data_2015 <- left_join(acs_tract_pop15, transit_trips_tracts_2015, by = "GEOID")
+#Test with anti_join to make sure it worked properly
+#stopifnot(
+#  anti_join(acs_tract_pop15, transit_trips_tracts_2015, by = "GEOID") %>%
+#    nrow() == 0
+#)
+# Error - therefore, check how many missing trips values for observations where we have population
+#missing_count <- sum(is.na(trips_data_2015$transit_trips_80ami))
+#cat("Number of missing values for transit_trips_80ami:", missing_count, "\n")
+# 810 missing values - aligns with cost data for the same year, good
+
+############################################################################
+# FOURTH ONE - Trips in 2019
+############################################################################
+trips_data_2019 <- left_join(acs_tract_pop19, transit_trips_tracts_2019, by = "GEOID")
+#Test with anti_join to make sure it worked properly
+stopifnot(
```

Comment:
`stopifnot` code is still active

Reply:
removed

Reply:
This one is still not running all the way through

Reply:
Shoot.. running fine on my end, so maybe a package issue? Where does it error for you?

Reply:
Line 191 seems to be the problem

Reply:
Darn!! I still missed one! Thanks Rob, should be good now

Reply:
Were you able to get it to run all the way through? I am now getting errors on the `get_acs` section (an API issue not a code issue) so I'm not able to confirm the updated section works yet.

Reply:
Yes, runs through on my end